README.md

```diff
@@ -1,241 +1,296 @@
+##### Install low-level runtime libraries for your GPU (1/2)
 
-```
-SYCL_ENABLE_HOST_DEVICE=1 python -c "import dpctl; print(dpctl.get_devices())"
-```
+At this time, only Intel GPUs are supported.
 
-If you have an Intel GPU and it is not detected, check the
-following steps:
+###### Intel GPU runtime libraries
 
-- Make sure you have installed the [latest GPU drivers](https://dgpu-docs.intel.com/installation-guides/index.html)
+For Intel GPUs, two backends are available. You might want to install both of those,
+and test if one gives better performances.
 
-- On Linux, check that the i915 driver is properly loaded:
+🚧 TODO: write a guide on how to select the device and the backend in a python script.
 
-  ```
-  $ lspci -nnk | grep i915
-        Kernel driver in use: i915
-        Kernel modules: i915
-  ```
+- **Intel Opencl for GPU**: the intel opencl runtime can be installed following
+  [this link](https://github.com/intel/compute-runtime#installation-options).
 
-- On Linux, check that the current user is in the `render` group, for instance:
+  ⚠⚠⚠ **WARNING** ⚠⚠⚠: for Ubuntu (confirmed for `focal` and `jammy`) the `apt`-based
+  installation is broken, see https://github.com/IntelPython/dpctl/issues/887 .
 
-  ```
-  $ groups
-  myuser adm cdrom sudo dip plugdev render lpadmin lxd sambashare docker
-  ```
+   ⚠ Like whenever installing packages outside of official repositories, existing
+   workarounds might make your system unstable and are not recommended outside of a
+   containerized environment and/or for expert users.
 
-  If not, add it with `sudo adduser $USER render`, logout and log back in, and check
-  again.
+   To not alter the `apt`-based version tree too much and risk other compatibility
+   issues, the recommended workaround consists in identifying the version that is
+   officially supported by your OS (use [packages.ubuntu.com](https://packages.ubuntu.com/search?keywords=intel-opencl-icd&searchon=names))
+   then find the corresponding build from the [Intel release page on github](https://github.com/intel/compute-runtime/releases)
+   and follow the instruction from the release page, e.g for `jammy`:
 
-- Install recent versions of the [runtime libraries](https://github.com/intel/compute-runtime/releases) (not yet available
-  as conda packages)
+   ```bash
+    mkdir neo
+    cd neo
+    wget https://github.com/intel/compute-runtime/releases/download/22.14.22890/intel-gmmlib_22.0.2_amd64.deb
+    wget https://github.com/intel/intel-graphics-compiler/releases/download/igc-1.0.10840/intel-igc-core_1.0.10840_amd64.deb
+    wget https://github.com/intel/intel-graphics-compiler/releases/download/igc-1.0.10840/intel-igc-opencl_1.0.10840_amd64.deb
+    wget https://github.com/intel/compute-runtime/releases/download/22.14.22890/intel-opencl-icd-dbgsym_22.14.22890_amd64.ddeb
+    wget https://github.com/intel/compute-runtime/releases/download/22.14.22890/intel-opencl-icd_22.14.22890_amd64.deb
+    cd ../ && rm -Rf neo
```

Comment:
We need to add instructions to install packages with either `dpkg` or `apt` I think.